docs/src/wallets/transferring-assets.md

```diff
@@ -6,8 +6,16 @@ With `wallet.transfer` you can initiate a transaction to transfer an asset from
 {{#include ../../../examples/wallets/src/lib.rs:wallet_transfer}}
 ```
 
-You can also transfer assets to a contract via `wallet.force_transfer_to_contract`.
+You can transfer assets to a contract via `wallet.force_transfer_to_contract`.
 
 ```rust,ignore
 {{#include ../../../examples/wallets/src/lib.rs:wallet_contract_transfer}}
 ```
+
+For transferring assets to the base layer chain, you can use `wallet.withdraw_to_base_layer`.
+
+```rust,ignore
+{{#include ../../../examples/wallets/src/lib.rs:wallet_withdraw_to_base}}
+```
+
+The above example creates an `Address` from a string and converts it to a `Bech32Address`. Next, it calls `wallet.withdraw_to_base_layer` by providing the address, the amount to be transferred, and the transaction parameters. Lastly, to verify that the transfer succeeded, the relevant message proof is retrieved with `provider.get_message_proof,` and the amount and the recipient is verified.
```

Comment:
Oh, this _definitely_ needs to be in the docs. SDK users refer to the docs first and code second. 

Sometimes we have to fight entropy manually 🤷 

Though I'm not opposed to having something _like_ this in the code itself, in addition to keeping this here. But it would have to be a lot more concise and less "documentation-narrative-like," haha.

Reply:
Just to clarify: I think that obviously that it should appear in the docs, but it would be better in the `wallet_withdraw_to_base` snippet in the form of comments.

Reply:
I've added some more comments to the snippet, but also kept the above description. Lmk if you are happy with that.